src/test/scala/com/workshop/RollingWindowThrottlerTest.scala

```diff
@@ -1,20 +1,24 @@
 package com.workshop
 
 import org.specs2.mutable.SpecificationWithJUnit
+import org.specs2.specification.Scope
 
 import scala.util.{Failure, Success, Try}
 
 class RollingWindowThrottlerTest extends SpecificationWithJUnit{
 
+  class ThrottlerScope extends Scope{
+    val anIp = "192.168.2.1"
+    val aThrottler = new RollingWindowThrottler(max = 1)
```

Comment:
i actually liked the fact that the `max` was defined in the test. maybe have a def you can override?